### ISSUE TITLE
Replace a File in File Manager

### ISSUE THREAD

**Issue by carlos22** [impresspages/ImpressPages#661]
In a lot of cases you want to replace for example a PDF file without changing the URL. I thik it would be rather easy to implement.

**Comment by maskas** [comment]
Good point. There could be a 'replace' button at the bottom of the window when you select an asset in the repository. Right?

**Comment by carlos22** [comment]
Yeah. Since you can only relplace one at a time, the button/link could be also somewhere near the image name (maybe an icon). But as you have the delete button at the bottom i think it makes sense to put the replace button also there...

**Comment by patrickruetter** [comment]
:thumbsup:
